### ISSUE TITLE
Signal Team Started working on a Pure Rust Implementaion for Signal Protocol!

### ISSUE THREAD

**Issue by shekohex** [Michael-F-Bryan/libsignal-protocol-rs#65]
See here: https://github.com/signalapp/libsignal-protocol-rust 

That's good news 😃 !

**Comment by rubdos** [comment]
Yes, we at Whisperfish also noticed, that's indeed great news. When it gets a bit of a stable interface, we could move the service crate over, and archive this repository... Bit of a shame that this was just finished now...
